src-self-hosted/translate_c.zig

```diff
@@ -363,7 +363,7 @@ fn prepopulateGlobalNameTable(ast_unit: *ZigClangASTUnit, c: *Context) !void {
     }
 }
 
-fn declVisitorNamesOnlyC(context: ?*c_void, decl: *const ZigClangDecl) callconv(.C) bool {
```

Comment:
I thought this might be causing CI test failures on windows/macos. This doesn't seem to be the case.